### PR TITLE
Adds enable/disable functions

### PR DESCRIPTION
**Pull request by ludeeus**
This adds 2 new functions to this package:
- Disable -- _Disable DNS blocking for a duration in seconds, or permanently._
- Enable -- _Enable DNS Blocking._

Go to `/admin/settings.php?tab=api` and click the "Show API token" button to get the `api_token`

### CODE REVIEW

**Comment by ludeeus** [comment]
Should I add a property extra here that contains the responses from the API call?
disable:

```json
{"status":"disabled","FTLnotrunning":true}
```

enable:
```json
{"status":"enabled","FTLnotrunning":true}
```

**Comment by ludeeus** [comment]
I forgot about this 😄 @fabaff any more you want me to do with this?

File: hole/__init__.py [hunk 1]
```diff
@@ -49,6 +50,48 @@ def __init__(self, host, loop, session, location='admin', tls=False,
...
+            _LOGGER.error(msg)
+            raise exceptions.HoleConnectionError(msg)
+
+
+    async def disable(self, duration=True):
+        """Disable DNS blocking on a *hole instance."""
+        if self.api_token is None:
+            _LOGGER.error("You need to supply an api_token to use this")
+            return
+        api = "{}?disable={}&auth={}".format(self.base_url, duration,
```
Comment (fabaff) [inline hole/__init__.py]: Can we use `params` here? Would be simpler to read.

Comment (ludeeus) [reply hole/__init__.py]: Moved params to params now :)

**Comment by fabaff** [comment]
Thanks. I will create a new release soon.

**Pull request merged**